<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Setting Device</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>87</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>BaudRate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>DataBits:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>Parity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>StopBits:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_29">
        <property name="text">
         <string>Timeout:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>QueryMode:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="HLed" name="led1" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="portBox1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="baudRateBox1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="dataBitsBox1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="parityBox1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="stopBitsBox1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="timeoutBox1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QComboBox" name="queryModeBox1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="openCloseButton1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Open/Close</string>
     </property>
    </widget>
    <widget class="HLed" name="led1_2" native="true">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>80</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Led</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>70</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Turn On/Turn Off</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Control</string>
    </attribute>
    <widget class="QPlainTextEdit" name="recvEdit1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>571</width>
       <height>141</height>
      </rect>
     </property>
     <property name="maximumBlockCount">
      <number>800</number>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="sendEdit1">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>0</y>
       <width>211</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sendButton1">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>190</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbModulation2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>77</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Mod</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Demod</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cmbModulation1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>79</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>BPSK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QPSK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0QPSK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8PSK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8QAM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16QAM</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>40</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>191</y>
       <width>79</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Modulation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>160</y>
       <width>68</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Data Rate</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneDataRate1">
     <property name="geometry">
      <rect>
       <x>425</x>
       <y>191</y>
       <width>146</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSet1">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Set MD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>585</x>
       <y>191</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Set DR</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Status</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HLed</class>
   <extends>QWidget</extends>
   <header>hled.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
